src/main/resources/permissions/mod-circulation-bff.csv

```diff
@@ -26,6 +26,8 @@ consortium-search.items.item.get
 circulation.check-in-by-barcode.post
 circulation-storage.loans.item.get
 circulation-storage.loans.collection.get
+circulation.loans.item.get
+circulation.loans.collection.get
```

Comment:
These permissions will be added to the system user by Okapi, should we add them for Eureka too?
Module descriptor section:
```
 "metadata": {
    "user": {
      "type": "system",
      "permissions": [
```